### ISSUE TITLE
PayPal Express `datePaid` is always null

### ISSUE THREAD

**Issue by engram-design** [craftcms/commerce-paypal#12]
See the below screenshot. PayPal Express `datePaid` is always null, despite payment going through, and being marked as paid. I can confirm payment has been indeed received in PayPal.

![screen shot 2018-11-23 at 12 31 43 pm](https://user-images.githubusercontent.com/1221575/48925789-d581a900-ef1b-11e8-8dfa-923ec6261b87.png)

Below is also an example order from the database.

Any ideas?

```
{
	"data":
	[
		{
			"billingAddressId": 7842,
			"shippingAddressId": 7843,
			"gatewayId": 2,
			"customerId": 17802,
			"id": 25592,
			"orderStatusId": 1,
			"number": "a72fbeff6773834aeb7a7c450d4580bb",
			"couponCode": null,
			"itemTotal": 5,
			"totalPrice": 5,
			"totalPaid": 5,
			"email": "web@web.com.au",
			"isCompleted": 1,
			"dateOrdered": "2018-11-23 01:55:04",
			"datePaid": null,
			"currency": "AUD",
			"paymentCurrency": "AUD",
			"lastIp": "::1",
			"orderLanguage": "en-AU",
			"message": null,
			"returnUrl": "/shop/customer/order?number=a72fbeff6773834aeb7a7c450d4580bb",
			"cancelUrl": "/shop/checkout/payment",
			"shippingMethodHandle": "metroDeliveryRegion",
			"dateCreated": "2018-11-23 01:44:08",
			"dateUpdated": "2018-11-23 01:55:07",
			"uid": "4352ad82-4418-482b-b27e-de6e6670c426",
			"paymentSourceId": null,
			"paidStatus": "paid",
			"reference": "a72fbef"
		}
	]
}
```

**Comment by engram-design** [comment]
@andris-sevcenko Anything I can do to help this one, or if it's verified as a bug?

**Comment by andris-sevcenko** [comment]
This was fixed with craftcms/commerce#600 and is pending a release. Sorry for the lack of updates on this one!

**Comment by engram-design** [comment]
Oh, easy then! Thanks @andris-sevcenko
